packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -252,6 +278,16 @@ export default async (homeP, endowments) => {
     return JSON.parse(feedPolicy);
   };
 
+  if (update) {
```

Comment:
This isn't really a concern of _init_ fast-usdc, right?

Please have a separate script or include a comment here and at the top explaining why not.

Reply:
I could rename it to `fast-usdc.builder.js`.

I'll try a separate script first to see how much code duplication there is.

Reply:
_init_ is a misnomer anyway.

 - #10490

Reply:
separate script is ~60 lines... about half is duplicated. Not so much that it's worth coupling the two purposes.